tests/test_basic.py

```diff
@@ -511,18 +519,26 @@ def test_rasterize_to_pixels(test_data, channels: int):
         isect_offsets,
         flatten_ids,
         backgrounds=backgrounds,
+        distloss=distloss,
     )
     torch.testing.assert_close(render_colors, _render_colors)
     torch.testing.assert_close(render_alphas, _render_alphas)
+    torch.testing.assert_close(render_distloss, _render_distloss)
 
     # backward
     v_render_colors = torch.randn_like(render_colors)
```

Comment:
Not an issue, just a general question regarding the use of `torch.autograd.grad(outputs * v_outputs ,inputs)` in the gradient tests. Why do we scale outputs by these random `v_outputs` weights? Shouldn't `torch.autograd.grad(outputs, inputs)` suffice for gradient checking?

Reply:
Makes sense, thanks. Here is a small visual of the dist loss as a heatmap whilst training: 
<img src=https://github.com/nerfstudio-project/gsplat/assets/30566358/fcd25e16-158d-4af0-bb79-ea7c595ca776 alt="dist loss" width = 400>

Reply:
Very cool color theme! What theme are you using here?

The reason I haven't merge this PR is that I didn't find this distortion loss to be helpful on ANY of the scenes in MipNeRF 360, which is a bit frustrated and not expected. 

Let me know if you find anything otherwise!

Reply:
I'm guessing the reason that this distortion loss isn't useful might have something to do with the imperfect depth accumulation formulation that we are currently using. The depth formulation from [Gaussian Opacity Fields](https://niujinshuchong.github.io/gaussian-opacity-fields/) or [RaDe-GS](https://baowenz.github.io/radegs/) seems to be more accurate, which might be a prerequisite for distortion loss to work.

Reply:
I tried this distortion loss with 2DGS and below are the results:
(with distortion loss)
![IMG_0672](https://github.com/nerfstudio-project/gsplat/assets/91100968/d6bd81fe-8204-4974-81e4-d93cdd1c5866)

(without distortion loss)
![IMG_0673](https://github.com/nerfstudio-project/gsplat/assets/91100968/bf4f0f99-19cb-4c46-b0b8-9c1228a18684)

Seems like the improvement is very subtle and focus on details; and this is also the case for the original 2DGS implementation; I will try the L2 version in the 2DGS paper next

Reply:
For those like me who were wondering what the awesome color scheme was, I think I tracked it down to `cmr.voltage`. https://cmasher.readthedocs.io/user/sequential/voltage.html